addons/web_tour/static/src/js/tip.js

```diff
@@ -107,6 +107,9 @@ var Tip = Widget.extend({
         this._unbind_anchor_events();
         clearTimeout(this.timerIn);
         clearTimeout(this.timerOut);
+        // clear this timeout so that we won't call _updatePosition after we
+        // destroy the widget and leave an undesired a bubble.
+        clearTimeout(this._transitionEndTimer);
```

Comment:
@aab-odoo can you or someone of the team, fast check this commit. Thanks

Reply:
I agree with the change